### PR TITLE
add owner/group/mode settings to gluster_mount LWRP

### PR DESCRIPTION
**Pull request by joyofhex**
I've also added a small amount of documentation describing the available options.

### CODE REVIEW

**Comment by joyofhex** [comment]
Hmm. I'm not positive that I even need the conditionals here. Removing them would fix this rubocop check, and seems to continue to work. However, I'm not confident that is correct chef code.

**Comment by joyofhex** [comment]
After checking with folks who understand chef much more than me I now think I can remove these guard conditionals (as these default to `nil` and that is the same default in the chef directory resource). Pushed that.

File: README.md [hunk 1]
```diff
@@ -65,6 +65,20 @@ gluster_mount 'volume_name' do
 end
 ```
 
+#### Parameters
+
+- `server` - The primary server to fetch the volfile from. Required.
+
+- `backup_server` - Backup servers to obtain the volfile from. Required.
```
Comment (shortdudey123) [inline README.md]: > `Required` 

Not per the [LWRP resource](https://github.com/joyofhex/chef-gluster/blob/mount_owner/resources/mount.rb#L29)

File: README.md [hunk 2]
```diff
@@ -65,6 +65,20 @@ gluster_mount 'volume_name' do
 end
 ```
 
+#### Parameters
```
Comment (shortdudey123) [inline README.md]: Should be level 3 not 4

**Comment by shortdudey123** [comment]
Looks good other than the comments above, thanks!

**Comment by joyofhex** [comment]
Thanks for catching those. Updated the PR.

**Pull request merged**